repair/row_level.cc

```diff
@@ -56,6 +56,8 @@ struct shard_config {
     sstring partitioner_name;
 };
 
+static bool inject_rpc_stream_error = false;
+
 distributed<db::system_distributed_keyspace>* _sys_dist_ks;
```

Comment:
This won't compile on its own (unused variable), please merge with the user.

Reply:
fixed